src/components/SubPages.js

```diff
@@ -74,18 +84,46 @@ function Line({ line, navigation }) {
             );
           } else if (char.Text && !char.link) {
             const isRegex = char.Text;
+
             if (!isRegex.match(regex)) {
+
+            // Etusivun otsikko kovakoodi
+            //
+              if (i == 10 && char.Text.includes("Teksti-TV")) {
```

Comment:
On olemassa varmasti parempi tapa muokata vain "Teksti-TV" otsikkoa kuin käyttää varia, mutta toimii näin.